Remotion/Web/Core/UI/SmartPageImplementation/SmartPageInfo.cs

```diff
@@ -274,16 +272,7 @@ private void Page_Init (object? sender, EventArgs e)
           _page.Header.Controls.AddAt(0, new HtmlHeadContents());
       }
 
-      HtmlHeadAppender.Current.RegisterUtilitiesJavaScriptInclude();
-
-      var resourceUrlFactory = SafeServiceLocator.Current.GetInstance<IResourceUrlFactory>();
-
-      var smartNavigationUrl = resourceUrlFactory.CreateResourceUrl(typeof(SmartPageInfo), ResourceType.Html, c_smartNavigationScriptFileUrl);
-      HtmlHeadAppender.Current.RegisterJavaScriptInclude(s_smartNavigationScriptKey, smartNavigationUrl);
-
-      var scriptUrl = resourceUrlFactory.CreateResourceUrl(typeof(SmartPageInfo), ResourceType.Html, c_scriptFileUrl);
-      HtmlHeadAppender.Current.RegisterJavaScriptInclude(s_scriptFileKey, scriptUrl);
-
+      HtmlHeadAppender.Current.RegisterWebClientScript();
```

Comment:
Consider moving this and the styleUrl to PreRenderSmartPage (see WxePageInfo). The registration during onInit was done decades ago but is no longer state of the art.